internal/sbi/producer/nf_management.go

```diff
@@ -517,8 +517,48 @@ func NFRegisterProcedure(
 	}
 }
 
+func configureNotifNfProfile(NotifProfile *models.NfProfileNotificationData, nfProfile models.NfProfile) {
```

Comment:
func configureNotifNfProfile(NotifProfile *models.NfProfileNotificationData, nfProfile *models.NfProfile) {

Reply:
param `NotifProfile` rename to `notifProfile`